core/urls.py

```diff
@@ -5,5 +5,5 @@
 app_name = "core"
 urlpatterns = [
     path("", views.index, name="index"),
-    path("profile/", views.profile, name="profile"),
+    path("panel/profile/<str:name>", views.profile, name="profile"),
```

Comment:
I think profile viewer is not part of a panel. If it is, then what isn't?

Reply:
Panel = navigation. So everything for logged users is inside panel